<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mesurerpage</class>
 <widget class="QDialog" name="mesurerpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(25, 25, 25)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Mesure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butMesure">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
background-color:rgb(48, 48, 48)</string>
   </property>
   <property name="text">
    <string>Lancer la mesure</string>
   </property>
  </widget>
  <widget class="QLabel" name="labGif">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labZ">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Z =</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="yLcd">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 48, 48);
color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labX">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>X =</string>
   </property>
  </widget>
  <widget class="QLabel" name="labY">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Y =</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="xLcd">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>240</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 48, 48);
color : white;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="zLcd">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>300</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48, 48, 48);
color : white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butSuivant">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>360</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
background-color:rgb(48, 48, 48)</string>
   </property>
   <property name="text">
    <string>Suivant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butDeco">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : white;
background-color:rgb(48, 48, 48)</string>
   </property>
   <property name="text">
    <string>Deconnexion</string>
   </property>
  </widget>
  <widget class="QLabel" name="labLogo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
